test/specs/basic/ignore-joiners.js

```diff
@@ -0,0 +1,36 @@
+/*!***************************************************
+ * mark.js
+ * https://github.com/julmot/mark.js
+ * Copyright (c) 2014–2016, Julian Motz
+ * Released under the MIT license https://git.io/vwTVl
+ *****************************************************/
+"use strict";
+describe("basic mark with ignore joiners", function () {
+    var $ctx1, $ctx2;
+    beforeEach(function (done) {
+        loadFixtures("basic/ignore-joiners.html");
+
+        $ctx1 = $(".basic-ignore-joiners > div:nth-child(1)");
+        $ctx2 = $(".basic-ignore-joiners > div:nth-child(2)");
+        new Mark($ctx1.get()).mark("Lorem ipsum", {
+            "ignoreJoiners": true,
```

Comment:
Please add `separateWordSearch: false` here. I know, it doesn't make a difference. Just to keep it consistent.